app/controllers/api/base_controller/parser.rb

```diff
@@ -72,16 +72,20 @@ def validate_api_parameters
       def parse_href(href)
         if href
           path = href.match(/^http/) ? URI.parse(href).path.sub(/\/*$/, '') : href
-          path = "#{@prefix}/#{path}" unless path.match(@prefix)
+          path = "#{prefix}/#{path}" unless path.match(prefix)
           path = path.sub(/\/*$/, '')
           return href_collection_id(path)
         end
         [nil, nil]
       end
 
+      def prefix
```

Comment:
This method name feel too generic.  Also, can't tell from the diff, but I think this should be private if not already.